lib/editor/result.js

```diff
@@ -0,0 +1,281 @@
+'use babel'
+import {CompositeDisposable, Emitter} from 'atom'
+import ResultView from './result-view'
+import fastdom from 'fastdom'
+
+let layers = {}
+
+export default class Result {
+  constructor (editor, lineRange, opts) {
+    this.emitter = new Emitter()
+    this.disposables = new CompositeDisposable()
+
+    this.editor = editor
```

Comment:
No, but I don't think we need to. If the editor is destroyed, so is the `MarkerDisplayLayer` and the `Decoration`s inside, which should in turn get rid of the view as well and therefore the `editor` reference. I hope...